<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="set_start_btn">
       <property name="text">
        <string>select start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="set_end_btn">
       <property name="text">
        <string>select end</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="algo_cbx">
       <item>
        <property name="text">
         <string>Manhattan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Euclidean</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chebyshev</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BFS</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start_btn">
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_btn">
       <property name="text">
        <string>clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>561</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>row:</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="row_edit">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>col:</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="col_edit">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>size:</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="size_edit">
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>density:</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="density_edit">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>interval:</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="interval_edit">
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="random_btn">
       <property name="text">
        <string>random</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="paint_btn">
       <property name="text">
        <string>paint</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>set_end_btn</tabstop>
  <tabstop>algo_cbx</tabstop>
  <tabstop>start_btn</tabstop>
  <tabstop>clear_btn</tabstop>
  <tabstop>row_edit</tabstop>
  <tabstop>col_edit</tabstop>
  <tabstop>size_edit</tabstop>
  <tabstop>density_edit</tabstop>
  <tabstop>paint_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
